### ISSUE TITLE
RN 0.61.2 Android handle element is not working (1.0.0-alpha.14)

### ISSUE THREAD

**Issue by kitfall** [osdnk/react-native-reanimated-bottom-sheet#103]
In iOS all is working very fine. When I compile my project for android the handle element dont acting. There isn't throwing any error or else, also when I setting initialSnap to 1 the layer is showing correctly, only the touch handler isn't working.
Is there a known issue that android doesent work? I've already tried with the example code (Collapsable.js) in iOS it is working, in android not.

Many Thanks

**Comment by cesar3030** [comment]
Is the handle element you're talking about the bottom sheet header(elements in renderHeader props)? I'm using RN61.1 it's working perfectly on iOS but I have issues with android. Sometimes, the header won't be render so just the renderContent is visible on the screen.

**Comment by kitfall** [comment]
Yes, the header element is visible and also in the render props. It seems only that the element dont receive any events (only in android). I doesn't think so that, that it is a problem of the new RN version, more an android problem.

`                <BottomSheet
                  ref={ref => this.bottomSheet = ref}
                  snapPoints={[40, dimension.width2_3 + 40]}
                  renderContent={this.renderBannerContent}
                  renderHeader={this.renderBannerHeader}
                  initialSnap={0}
                  enabledInnerScrolling={false}
                  callbackNode={this.sheetOpacity}
                />`

![alt text](https://i.ibb.co/XzYydxF/android.jpg)

**Comment by cesar3030** [comment]
Ok we don't have the same issue then.

**Comment by romanonthego** [comment]
It's a may be a bug with gesture-handler itself - when rendered __inside `react-native` `<Modal/>`__ touch/swipe etc events does not register for any components in `gesture-handler` library (even buttons). Android only, tracked here - https://github.com/kmagiera/react-native-gesture-handler/issues/139.

Or, if bug persisted when rendered outside the modal, well, it's something else.

**Comment by harrisrobin** [comment]
Having the same issue as well.

**Comment by hungdt-ibl** [comment]
PanGestureHandler can not work on android.

**Comment by imdanielpiva** [comment]
I think this issue is releated https://github.com/osdnk/react-native-reanimated-bottom-sheet/issues/91! Just commented a more info about what I could get there.

**Comment by imdanielpiva** [comment]
How did you manage to solve this, just downgrading from `1.0.0-alpha.14`?

**Comment by AlexeyJersey** [comment]
Any update?

**Comment by walterholohan** [comment]
I am having the same issue as above. On Android the PanGestureHandler does not seem to work if the `<BottomSheet />` rendered inside a `react-native` `<Modal />` component. Has anyone found a workaround for this?

It seems that if you have this inside a `react-navigation` stack navigator it doesn't work if after https://github.com/software-mansion/react-native-gesture-handler/pull/937 was merged.

Example:
```
<Modal animationType="none" transparent onRequestClose={onCancel} visible={visible}>
      <TouchableWithoutFeedback onPress={onCancel}>
        <Animated.View
          style={[
            styles.modal,
            {
              opacity: backgroundOverlayOpacity,
            },
          ]}
        />
      </TouchableWithoutFeedback>
      <BottomSheet
        ref={bottomSheetRef}
        callbackNode={callbackNode.current}
        onCloseEnd={onCloseEnd}
        snapPoints={[0, HEIGHT]}
        enabledContentTapInteraction={false}
        renderContent={() => (
          <View />
        )}
      />
    </Modal>
```
